src/main/java/com/dabsquared/gitlabjenkins/GitLabWebHook.java

```diff
@@ -108,6 +108,14 @@ public void run() {
         while (restOfPathParts.hasNext()) {
             paths.add(restOfPathParts.next());
         }
+        
+        // remove everything till we found 'commits'
+        for (Iterator<String> it = paths.iterator(); it.hasNext();) {
+            String path = it.next();
+            if (path.equals("commits"))
```

Comment:
after a second look I've missed that the path could also hold "build" and "builds". I'll create a new PR with a better approach tomorrow.